common/src/main/java/org/corfudb/common/CorfuSourceCodeVersion.java

```diff
@@ -0,0 +1,35 @@
+package org.corfudb.common;
+
+
+/**
+ * Static class for representing the corfu source code version using git commit id.
+ */
+public final class CorfuSourceCodeVersion {
+
+    /**
+     * Original git commit id of the head of master branch, which should be updated upon every
+     * corfu artifact deployment.
+     */
+    private static final String CORFU_SOURCE_VERSION_STRING = "d42d17b360acfd2eccac4862b8f1fd1a79f75586";
```

Comment:
nit: can we rename it to `CORFU_SOURCE_CODE_VERSION_STRING` to keep it consistent with the class name?